File: 6ku/CreatePhoneNumber/CreatePhoneNumber.py
def create_phone_number(n):
    n = "".join([str(i) for i in n])
    return f"({n[0:3]}) {n[3:6]}-{n[6:10]}"

# best
def create_phone_number1(n):
    print(*n)
	# return "({}{}{}) {}{}{}-{}{}{}{}".format(*n)

File: 6ku/CreatePhoneNumber/CreatePhoneNumber.js
function createPhoneNumber(numbers){
    return `(${numbers.slice(0, 3).join('')}) ${numbers.slice(3, 6).join('')} - ${numbers.slice(6).join('')}`;
}

// best 
function createPhoneNumber(numbers){
    return numbers.join('').replace(/(...)(...)(.*)/, '($1) $2-$3');
  }